velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -83,67 +111,37 @@ TEST_P(MultiAggregatesTest, singlePartitionWithSortOrders) {
 }
 
 TEST_P(MultiAggregatesTest, randomInput) {
-  auto vectors = makeFuzzVectors(
-      ROW({"c0", "c1", "c2", "c3"},
-          {BIGINT(), SMALLINT(), INTEGER(), BIGINT()}),
-      10,
-      2);
-  createDuckDbTable(vectors);
-
-  std::vector<std::string> overClauses = {
-      "partition by c0 order by c1, c2, c3",
-      "partition by c1 order by c0, c2, c3",
-      "partition by c0 order by c1 desc, c2, c3",
-      "partition by c1 order by c0 desc, c2, c3",
-      "partition by c0 order by c1",
-      "partition by c0 order by c2",
-      "partition by c0 order by c3",
-      "partition by c1 order by c0 desc",
-      "partition by c0, c1 order by c2, c3",
-      "partition by c0, c1 order by c2",
-      "partition by c0, c1 order by c2 desc",
-      "order by c0, c1, c2, c3",
-      "partition by c0, c1, c2, c3",
+  auto size = 50;
+  std::vector<VectorPtr> vectors;
```

Comment:
This could be auto rowVector = makeRowVector({
makeFlatFuzzVector(),
makeFlatVector<>...
makeFlatVector<>...
makeFlatFuzzVector...
})

The extra vectors argument with the emplace_back is not needed.